src/controllers/authController.ts

```diff
@@ -0,0 +1,35 @@
+import { isEmpty, pool } from "../utils";
+import { Member, MemberSchema } from "../types/types";
+import { z } from "zod";
+import { RowDataPacket } from "mysql2";
+
+
+class AuthController {
+  async getMember(id: string): Promise<Member | Error> {
+    const [memberInfo] = await pool.query(
+      `SELECT * FROM Member WHERE uuid = ?`,
+      [id]
+    )
+
+    try {
+      const member = (memberInfo as RowDataPacket[])[0]
+      const typedUser = MemberSchema.parse({
+        uuid: member.uuid, 
+        first_name: member.first_name, 
+        last_name: member.last_name, 
+        email: member.email, 
+        active_member: !!member.active_member, 
```

Comment:
ah are these double exclamation marks to handle the fact the database returns 0/1 instead of true/false?

Reply:
yea